packages/react-core/src/components/Modal/examples/Modal.md

```diff
@@ -13,84 +13,114 @@ import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
 
 ## Examples
 
-### Basic
+### Basic modals
+
+Basic modals give users the option to either confirm or cancel an action. To flag an open modal, use the `isOpen` property. To execute a callback when a modal is closed, use the `onClose` property.
 
 ```ts file="./ModalBasic.tsx"
 ```
 
-### With description
+### Scrollable modals
+
+To enable keyboard-accessible scrolling of a modal’s content, pass `tabIndex={0}` to the `<Modal>`.
+
+```ts file="ModalWithOverflowingContent.tsx"
+```
+
+### With a static description
+
+To provide additional information about a modal, use the `description` property. Descriptions are static and do not scroll with other modal content.
 
 ```ts file="./ModalWithDescription.tsx"
 ```
 
 ### Top aligned
 
+To override a modal's default center alignment, use the `position` property. In this example, `position` is set to "top", which moves the modal to the top of the screen.
+
 ```ts file="./ModalTopAligned.tsx"
 ```
 
-### Small
+### Small modal
+
+To shorten the horizontal width of a modal, the `variant` property can be set to "small".
 
 ```ts file="./ModalSmall.tsx"
 ```
 
-### Medium
+### Medium modal
+
+To shorten the horizontal width of a modal, the `variant` property can be set to "medium".
 
 ```ts file="./ModalMedium.tsx"
 ```
 
-### Large
+### Large modal
+
+To shorten the horizontal width of a modal, the `variant` property can be set to "large".
 
 ```ts file="./ModalLarge.tsx"
 ```
 
 ### Custom width
 
+To choose a specific width for a modal, use the `width` property. This example has a `width` of "50%".
+
 ```ts file="./ModalCustomWidth.tsx"
 ```
 
 ### Custom header and footer
 
+To add a custom header and footer to a modal, set the `header` and `footer` properties to a custom implementation. The following example passes title components into both the header and the footer and also passes an icon to the footer.
+
 ```ts file="./ModalCustomHeaderFooter.tsx"
 ```
 
 ### No header or footer
 
+To exclusively present information in a modal, remove the `header` and/or `footer`.
+
 ```ts file="./ModalNoHeaderFooter.tsx"
 ```
 
 ### Title icon
 
+To add an icon before a modal’s title, use the `titleIconVariant`, which can be set equal one of the predefined variants -- "success", "danger", "warning", "info", and "default" -- or to an imported custom icon. The following example uses a "warning" variant.
```

Comment:
Last small nit! Otherwise LGTM

```suggestion
To add an icon before a modal’s title, use the `titleIconVariant`, which can be set to one of the predefined variants -- "success", "danger", "warning", "info", and "default" -- or to an imported custom icon. The following example uses a "warning" variant.
```